gammapy/maps/hpxsparse.py

```diff
@@ -138,11 +138,12 @@ def _make_cols(self, header, conv):
 
         elif len(shape) == 1:
             cols.append(fits.Column(conv.colname(indx=conv.firstcol),
-                                    'E', array=self.data.data.astype(float)))
+                                    'E', array=self.data[...].astype(float)))
```

Comment:
Why `[...]` here? That means "make a copy", no? Why make a copy? Do we do that anywhere else in map -> FITS, to decouple the map data from the FITS data? I thought we just pass it on without making a copy?

Reply:
The issue here is that the `data` member only returns non-zero elements by default.  The ellipsis is needed to force it to instantiate a dense array.